input_topic: "/image_synchroniser/grouped_images"
output_topic: "/linear_motion_compensation/aligned_images"
camera_index: 0 # Index of the camera in which images will be aligned. E.g. If image synchroniser outputs image array with 2 cameras, setting this to zero will perform motion compensation on the images of the first camera
reference_index: 1 # Align all images to the image with the specified image indice. E.g. If reference_indice is 1, all images will be aligned w.r.t the second image in the sequence
subsample_factor: 1.0 # Amount to subsample images before applying phase correlation image registration

